src/models/auth.rs

```diff
@@ -139,6 +139,16 @@ pub struct AuthToken {
     pub sub: UserId,
     pub device_id: DeviceId,
     pub jti: u32,
+    pub exp: u64,
+}
+impl AuthToken {
+    pub fn is_expired(&self) -> bool {
```

Comment:
I thought our jwt library checked this automatically in the decode fn? worth checking out.

Reply:
yeah, thanks, missed that